frontend/src/Components/Checkbox/Checkbox.tsx

```diff
@@ -23,7 +23,7 @@ export function Checkbox({ name, onClick, disabled, checked, alignment = 'left',
         disabled={disabled}
         checked={checked}
       />
-      <div className={styles.checkbox__box}></div>
+      <div className={disabled ? styles.checkbox__disabled : styles.checkbox__box}></div>
```

Comment:
Knyttet til andre kommentaren min

```suggestion
      <div className={classNames(styles.checkbox__box, disabled && styles.checkbox__disabled)}></div>
```

Reply:
Nå tror jeg de ble som du mener